Entity/Form/Search.php

```diff
@@ -353,4 +360,22 @@ public function setContentType($contentType)
         $this->contentType = $contentType;
         return $this;
     }
+
+    /**
+     * @return int
```

Comment:
Redundant comment as return type is defined allready

Reply:
Solved, but I would like to find a way to disable itnfrom php storm? https://www.jetbrains.com/help/phpstorm/settings-file-and-code-templates.html?

Reply:
I think the very last point in this article might help https://www.jetbrains.com/help/phpstorm/phpdoc-comments.html